qemu/tests/cfg/in_place_upgrade.cfg

```diff
@@ -53,6 +53,8 @@
         - compose_install:
             com_install = "yes"
             com_ins_leapp = "yum install leapp-upgrade -y"
+        - stream_install:
+            # Please modify the build in the internal cfg
```

Comment:
I think this should add an example rather than point to internal.

Reply:
updated